### PR TITLE
ISV-4799: update vault preflight-decryption-key-pub

### PR DESCRIPTION
**Pull request by johnbe11**
(related to recent CI security breach)

### CODE REVIEW

**Comment by johnbe11** [comment]
@Allda PTAL

**Comment by johnbe11** [comment]
@mporrato PTAL

File: ansible/vaults/prod/preflight-decryption-key-pub [hunk 1]
```diff
@@ -1,52 +1,52 @@
...
+l/4LFHbU1w8HbU97kxXslhqF5yhIxUgVBsl1kJWBjhJ2T2CE3Ksq5BWv2TXGOrbA
+xS7FCTZewBb/yVr7ewettOYHq/w4VjhxhuAUREKGCAuQqlSlYaPO14x6yx5//uJH
+vAG/BZ5e0tETUJy+a1pBizbUPAXc4Riet2tBBrz3ZurEvlTpfcvn5AnZTN6d7mNw
+CIaLIBuMGSYCdrJE5aR3Z0I3+57NWAnbgHQhPxp3r3pSifGDRgk22Ofy+juQIoYQ
+6sXh9DZ9TGkTA3/LVa0hL3q4t4KUOnjBvjbQk84FxrBpVi03ErFIaBkzuuIg2DBy
+bV+Y3lzY/6h7M3x2GER8hTOop6bAlks5Y1xHWqF1QZ80bCcNjR7exP7jv87e9KVZ
+d8oTjtxM64Wk7B/7ClyQh0nv/k2GLsEAZiGYBpkqoPLGxetVN+TLpGNrghTVNe65
+bB8trK4vNHUnObmAdIzzUmq0lmZmxFzZ
+=QQff
+-----END PGP PUBLIC KEY BLOCK-----
```
Comment (Allda) [inline ansible/vaults/prod/preflight-decryption-key-pub:52]: There is a missing line at the end of the file. Please add it.

**Review by Allda — APPROVED** [review]
Appart from one minor comment the change looks good.

**Comment by johnbe11** [comment]
@Allda updated

**Review by mporrato — APPROVED** [review]

**Pull request merged**